website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another Windows Subsystem for Linux (WSL) distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, [Podman Desktop creates a Windows Subsystem for Linux (WSL) virtual machine: the Podman Machine](/docs/podman/creating-a-podman-machine.md).
+It also configures the Windows Podman client to communicate with the Podman Machine.
+However, it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the most common context to walk you through the steps to configure your WSL distribution:
+
+- Ubuntu distribution of Linux.
+- Default Podman Machine.
+
+In foldable details, you can find alternative steps for least common contexts:
+
+- Custom WSL distribution.
+- Custom Podman Machine.
+
+## Configuring your WSL distribution
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. To communicate with the remote Podman Machine, you need a Podman client.
+
+   To benefit from the latest features, such as `podman kube` subcommands, and to enable the remote mode by default, consider installing the `podman-remote` binary rather than the `podman` package from the distribution.
+
+   Download the `podman-remote` binary from the [Podman releases page](https://github.com/containers/podman/releases/latest).
+
+1. Configure the Podman client in your WSL distribution to communicate with the remote Podman machine defined by Podman Desktop.
+
+   This will ensure consistency when you are working with Podman from all your different environments
+
+   Set the default Podman system connection to your Podman Machine:
+
+   ```shell-session
+   $ podman system connection add --default podman-machine-default-root unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock
```

Comment:
This seems wrong to default to Podman rootful because if you follow this through, you will not be able to view the containers from Podman Desktop (I actually thought the integration was broken, until I dug deeper). I recommend to replace `podman-root` with `podman-user`

Reply:
An additional advantage to starting with user instead of root is that the steps for adding permissions to the socket for GID 10 are no longer necessary (at least for Ubuntu)

Reply:
Following previous suggestions, I removed mention to rootful/rootless Podman in the default user story. This distinction is implicitly present in the details right after. I understand I should add it again there.

Reply:
Unfortunately, the default changed from rootless to rootful to address issues specific to the rootless mode, notably when using Kind. It hopefully might change back later.